lib/test-config.js

```diff
@@ -1,5 +1,12 @@
 'use strict'
 
 module.exports = {
+  tested: {
+    // protocol from https://nodejs.org/docs/latest-v8.x/api/url.html#url_urlobject_protocol
+    // is followed by a colon
+    protocol: 'http:',
+    hostname: 'localhost',
+    port: 1111,
```

Comment:
I see that this is trying to make things more DRY, though I wonder if it would be clearer to put the URL here, and also more consistent with the other URLs we configure. I also think this could be loaded from the environment, as it's a pretty unusual case.

Something like:

```js
liveServerUrl: process.env.LIVE_SERVER_URL
```

Reply:
Good idea. Done.